neurodsp/utils/core.py

```diff
@@ -17,15 +17,17 @@ def get_avg_func(avg_type):
 
     Returns
     -------
-    avg_func : callable
-        Requested averaging function.
+    func : callable
+        Requested function.
     """
 
-    check_param_options(avg_type, 'avg_type', ['mean', 'median'])
+    check_param_options(avg_type, 'avg_type', ['mean', 'median', 'sum'])
 
     if avg_type == 'mean':
-        avg_func = np.mean
+        func = np.mean
     elif avg_type == 'median':
-        avg_func = np.median
+        func = np.median
+    elif avg_type == 'sum':
+        func = np.sum
```

Comment:
sum could be added to the docstring:  `{'mean', 'median', 'sum'}`

Reply:
Oops, good catch! Fixed!